script/fmt

```diff
@@ -6,7 +6,7 @@ hash goimports 2>/dev/null && {
 }
 
 # don't run gofmt in these directories
-ignored=(/bin/ /docs/ /log/ /man/ /tmp/ /vendor/ /rpm/ /docker/ /debian/)
+ignored=(/bin/ /docs/ /log/ /man/ /tmp/ /vendor/ /rpm/ /docker/ /debian/ /src/)
```

Comment:
Is this `src/` directory used by AppVeyor as a workspace to build the Windows installer? If so, perhaps it would be good to document this above:

```bash
# don't run gomt in these directories...
#  - `src/`: used by AppVeyor to ...
```